src/picker_gui.h

```diff
@@ -175,10 +177,14 @@ class PickerWindow : public PickerWindowBase {
 	static const int PREVIEW_LEFT = 31; ///< Offset from left edge to draw preview.
 	static const int PREVIEW_BOTTOM = 31; ///< Offset from bottom edge to draw preview.
 
+	static const int STEP_PREVIEW_HEIGHT = 16; ///< Step for decreasing or increase preivew button height.
+	static const int MAX_PREVIEW_HEIGHT = 48 * 3; ///< Maximum height of each preview button.
```

Comment:
Does this not work? (Maybe needs to be `constexpr`?) Not a big deal, just cleaner.

```suggestion
	static const int MAX_PREVIEW_HEIGHT = PREVIEW_HEIGHT * 3; ///< Maximum height of each preview button.
```